src/report/report.controller.ts

```diff
@@ -45,6 +50,24 @@ export class ReportController {
     return this.reportService.createReport(createReportDto, images);
   }
 
+  @ApiCreatedResponse({
+    description: 'New Report has been successfully sent',
+    type: Message,
+  })
+  @ApiConsumes('multipart/form-data')
+  @UseInterceptors(FilesInterceptor('images', 4))
+  @Post('/bark-beetle')
+  sendBarkBeetleReport(
+    @Body() createReportDto: CreateReportDto,
+    @UploadedFiles() images: Array<Express.Multer.File>,
+  ): Promise<MessageSendingResponse> {
```

Comment:
Not sure if it's really good to return such response to public. It might contain some confidential information.